dbt/adapters/athena/impl.py

```diff
@@ -153,6 +155,38 @@ def clean_up_table(self, database_name: str, table_name: str):
     def quote_seed_column(self, column: str, quote_config: Optional[bool]) -> str:
         return super().quote_seed_column(column, False)
 
+    @available
+    def upload_seed_to_s3(
+        self,
+        s3_data_dir: Optional[str],
+        s3_data_naming: Optional[str],
+        external_location: Optional[str],
+        database_name: str,
+        table_name: str,
+        table: agate.Table,
+    ) -> str:
+        conn = self.connections.get_thread_connection()
+        client = conn.handle
+
+        # TODO: consider using the workgroup default location when configured
+        s3_location = self.s3_table_location(s3_data_dir, s3_data_naming, database_name, table_name, external_location)
+        bucket, prefix = self._parse_s3_path(s3_location)
+
+        # Upload as json to support datetime parsing and better type inference
+        # OpenCSVSerde only supports UNIX timestamps
+        file_name = f"{table_name}.json"
```

Comment:
table name is in the path already, how about using a `uuid.uuid4()` instead?

Reply:
I'm already using `object_name` the line below combining the prefix and the actual `file_name` that we're uploading 😄

Reply:
Using a `uuid` as the file name would write a new file every time the seed command is run, which may be excessive? Having a consistent name overwrites the file?

Reply:
Sorry, late review time :D ignore the comment about filename...:D

> Using a uuid as the file name would write a new file every time the seed command is run, which may be excessive? Having a consistent name overwrites the file?
indeed using uuid means that we need to clean up the object before re-uploading, name overwrite as you did is maybe the best behavior.